### PR TITLE
Feature/remove gulp

### PR DESCRIPTION
**Pull request by brainthinks**

### CODE REVIEW

**Comment by brainthinks** [comment]
it updates all of the development tools we use (webpack, dev server, babel, eslint, etc) so that we can iterate on changes in seconds rather than minutes.

this isn't a code change that is meant to affect the player, it is meant to affect the development experience and environment.

File: README.md [hunk 1]
```diff
@@ -232,9 +232,25 @@ See the LICENSE file at the root of this repository.
 
 ## @todos
 
+* implement dev server
```
Comment (uhhlittle) [inline README.md]: you have already done this, so, remove it.

File: README.md [hunk 2]
```diff
@@ -48,28 +48,36 @@ Chrome 52+ is required to run this videojs extension.  All other browsers are cu
...
+
+#### Vagrant
+
+1. `cp scripts/deploy/Vagrantfile ..`
+1. `vagrant destroy -f && vagrant up && vagrant ssh`
+1. `cd /vagrant/clsp-videojs-plugin`
+1. `rm -rf node_modules`
+1. `sudo scripts/deploy/provision-bootstrap.sh`
+1. `yarn install`
+1. `yarn run serve:vagrant`
```
Comment (uhhlittle) [inline README.md]: provide Vagrant URL here plz

File: scripts/serve.sh [hunk 3]
```diff
@@ -2,10 +2,10 @@
...
 
-dps "Building..."
-npm run build
-ec "Successfully built!" "Failed to build!"
+# dps "Building..."
+# yarn run build
+# ec "Successfully built!" "Failed to build!"
 
-dps "Serving"
+dps "Brining up webpack dev server..."
```
Comment (uhhlittle) [inline scripts/serve.sh]: Bring*ing

**Pull request merged**